packages/ui/src/views/layouts/SandboxLayout.tsx

```diff
@@ -17,10 +19,7 @@ function Root({ children }: { children: React.ReactNode }) {
 function LeftPanel({ children, className }: { children: React.ReactNode; className?: string }) {
```

Comment:
Looks like there's a few references to `React` in this, but it's not imported, can we tidy up by importing `ReactNode`?